src/App.tsx

```diff
@@ -22,7 +27,43 @@ function App() {
 
   return (
     <>
-      <div>
+      <AppBar position="static">
+        <Toolbar>
+          <Typography variant="h6" color="inherit" component="div">
+            Invoices
+          </Typography>
+
+          <MenuItem>
+            <Typography textAlign="center">New Invoice</Typography>
+          </MenuItem>
+        </Toolbar>
+      </AppBar>
+
+      <InvoiceList></InvoiceList>
```

Comment:
Generally, when component doesn't accept children, the convention is to use self closing elements `<InvoicesList /> `

Reply:
corrected